Application/Items/Commands/Delete.cs

```diff
@@ -34,7 +34,7 @@ public async Task<Unit> Handle(Command request, CancellationToken cancellationTo
 
                 if (item == null)
                 {
-                    throw new NotFoundException(nameof(Item), request.Id);
+                    throw new NotFoundException($"Item {request.Id} does not exist");
```

Comment:
there's no point in changing this, as the 404 response already specifies that something does not exist